crates/data-generator/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "rjacraft_data_generator"
+description = "Data generator for Rjacraft."
+categories = ["development-tools::build-utils", "encoding"]
```

Comment:
This crate doesn't need metadata

Reply:
Not against removing it, but why? @JarvisCraft

Reply:
Because we don't need anyone finding this crate on crates.io. It's practically junk for other developers.

Reply:
The crate still may be usable by other developers (although the use cases are more exotic) so I am **for** keeping metadata.